src/site/layouts/support_resources_detail_page.drupal.liquid

```diff
@@ -63,6 +63,9 @@
         <!-- Line break -->
         <hr class="vads-u-margin-top--0" />
 
+        <!-- Tags -->
+        {% include "src/site/includes/tags.drupal.liquid" with fieldTags = fieldTags %}
```

Comment:
Since each entity in the `fieldTags.entity` is a `ParagraphAudienceTopics`,  I wonder if it would it make sense to add  the `entityBundle` property into `src/site/stages/build/drupal/graphql/paragraph-fragments/audienceTopics.paragraph.graphql.js` and then make the template at `src/site/paragraphs/audience_topics.drupal.liquid` so it follows the pattern of other paragraph types. But then again, then you would have to iterate from the layout and include the `audience_topics` paragraph template for each entity, so it's a little more verbose. 

IDK, what do you think? `includes` is fine, it just feels a little cluttered there

Reply:
I see what you mean 🤔 and I also see how that would add some verbosity to each LC template too, like 2-4 more lines for each one, I might keep it like this just for MVP 1.0 and then later on we can refactor if necessary, what do you think?

Reply:
That sounds good to me!